packages/common/src/create-icon-set.tsx

```diff
@@ -202,7 +204,8 @@ export function createIconSet<GM extends Record<string, number>>(
     }
 
     try {
-      const imagePath = await NativeIconAPI.getImageForFont(
+      // biome-ignore lint/style/noNonNullAssertion: this is checked in ensureGetImageAvailable
+      const imagePath = await NativeIconAPI!.getImageForFont(
```

Comment:
if `ensureGetImageAvailable` were to return `NativeIconAPI` there would be no need for non null assertion here and above